2021-10-31 06:57:45,296 INFO    MainThread:397 [wandb_setup.py:_flush():71] setting env: {}
2021-10-31 06:57:45,296 INFO    MainThread:397 [wandb_setup.py:_flush():71] setting login settings: {}
2021-10-31 06:57:45,296 INFO    MainThread:397 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '0e032cd7f7534093326d606d65729a6bb9c51025'}
2021-10-31 06:57:45,296 INFO    MainThread:397 [wandb_setup.py:_flush():71] setting login settings: {}
2021-10-31 06:57:45,296 INFO    MainThread:397 [wandb_init.py:_log_setup():357] Logging user logs to /home/ubuntu/w251-hw9/wandb/run-20211031_065745-1pgsa3v1/logs/debug.log
2021-10-31 06:57:45,296 INFO    MainThread:397 [wandb_init.py:_log_setup():358] Logging internal logs to /home/ubuntu/w251-hw9/wandb/run-20211031_065745-1pgsa3v1/logs/debug-internal.log
2021-10-31 06:57:45,296 INFO    MainThread:397 [wandb_init.py:_jupyter_setup():307] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fa09855bcd0>
2021-10-31 06:57:45,296 INFO    MainThread:397 [wandb_init.py:init():390] calling init triggers
2021-10-31 06:57:45,296 INFO    MainThread:397 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-10-31 06:57:45,296 INFO    MainThread:397 [wandb_init.py:init():435] starting backend
2021-10-31 06:57:45,296 INFO    MainThread:397 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-31 06:57:45,322 INFO    MainThread:397 [backend.py:ensure_launched():199] starting backend process...
2021-10-31 06:57:45,326 INFO    MainThread:397 [backend.py:ensure_launched():204] started backend process with pid: 449
2021-10-31 06:57:45,327 INFO    MainThread:397 [wandb_init.py:init():444] backend started and connected
2021-10-31 06:57:45,329 INFO    MainThread:397 [wandb_run.py:_label_probe_notebook():866] probe notebook
2021-10-31 06:57:45,329 INFO    MainThread:397 [wandb_run.py:_label_probe_notebook():876] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-10-31 06:57:45,329 INFO    MainThread:397 [wandb_init.py:init():503] updated telemetry
2021-10-31 06:57:45,343 INFO    MainThread:397 [wandb_init.py:init():533] communicating current version
2021-10-31 06:57:46,042 INFO    MainThread:397 [wandb_init.py:init():538] got version response 
2021-10-31 06:57:46,042 INFO    MainThread:397 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-10-31 06:57:46,139 INFO    MainThread:397 [wandb_init.py:init():576] starting run threads in backend
2021-10-31 06:57:46,645 INFO    MainThread:397 [wandb_run.py:_console_start():1693] atexit reg
2021-10-31 06:57:46,645 INFO    MainThread:397 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.WRAP
2021-10-31 06:57:46,646 INFO    MainThread:397 [wandb_run.py:_redirect():1604] Wrapping output streams.
2021-10-31 06:57:46,647 INFO    MainThread:397 [wandb_run.py:_redirect():1628] Redirects installed.
2021-10-31 06:57:46,647 INFO    MainThread:397 [wandb_init.py:init():603] run started, returning control to user process
2021-10-31 06:57:46,649 INFO    MainThread:397 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-31 06:57:46,649 INFO    MainThread:397 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-10-31 06:59:58,525 INFO    MainThread:397 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-31 06:59:58,528 INFO    MainThread:397 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-31 06:59:58,528 INFO    MainThread:397 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-10-31 07:00:08,399 INFO    MainThread:397 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-31 07:00:08,443 INFO    MainThread:397 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-31 07:00:08,443 INFO    MainThread:397 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-10-31 07:00:25,605 INFO    MainThread:397 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-31 07:00:25,605 INFO    MainThread:397 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-31 07:00:25,605 INFO    MainThread:397 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-10-31 07:00:30,034 INFO    MainThread:397 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-31 07:00:30,679 INFO    MainThread:397 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-31 07:00:30,679 INFO    MainThread:397 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-10-31 07:01:14,109 INFO    MainThread:397 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-31 07:01:14,756 INFO    MainThread:397 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-31 07:01:14,756 INFO    MainThread:397 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-10-31 07:01:26,940 INFO    MainThread:397 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-31 07:01:27,575 INFO    MainThread:397 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-31 07:01:27,575 INFO    MainThread:397 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-10-31 07:03:18,395 INFO    MainThread:397 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-31 07:03:18,396 INFO    MainThread:397 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-31 07:03:18,396 INFO    MainThread:397 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-10-31 07:05:30,928 INFO    MainThread:397 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-31 07:05:30,929 INFO    MainThread:397 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-31 07:05:30,930 INFO    MainThread:397 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-10-31 07:05:56,977 INFO    MainThread:397 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-31 07:05:56,986 INFO    MainThread:397 [wandb_setup.py:_flush():71] setting env: {}
2021-10-31 07:05:56,986 INFO    MainThread:397 [wandb_setup.py:_flush():71] setting login settings: {}
2021-10-31 07:05:56,986 INFO    MainThread:397 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '0e032cd7f7534093326d606d65729a6bb9c51025'}
2021-10-31 07:05:56,986 INFO    MainThread:397 [wandb_setup.py:_flush():71] setting login settings: {}
2021-10-31 07:05:56,987 INFO    MainThread:397 [wandb_init.py:_log_setup():357] Logging user logs to /home/ubuntu/w251-hw9/wandb/run-20211031_070556-2qn8ah21/logs/debug.log
2021-10-31 07:05:56,987 INFO    MainThread:397 [wandb_init.py:_log_setup():358] Logging internal logs to /home/ubuntu/w251-hw9/wandb/run-20211031_070556-2qn8ah21/logs/debug-internal.log
2021-10-31 07:05:56,987 INFO    MainThread:397 [wandb_init.py:init():390] calling init triggers
2021-10-31 07:05:56,987 INFO    MainThread:397 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {'epochs': 2, 'batch_size': 500, 'momentum': 0.9, 'WEIGHT_DECAY': 0.0001, 'arch': 'resnet18'}
2021-10-31 07:05:56,987 INFO    MainThread:397 [wandb_init.py:init():410] re-initializing run, found existing run on stack: 1pgsa3v1
2021-10-31 07:05:56,988 INFO    MainThread:397 [wandb_run.py:finish():1347] finishing run rochelleli/w251-hw9/1pgsa3v1
2021-10-31 07:05:56,989 INFO    MainThread:397 [jupyter.py:save_history():429] not saving jupyter history
2021-10-31 07:05:56,989 INFO    MainThread:397 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-10-31 07:05:56,989 INFO    MainThread:397 [wandb_init.py:_jupyter_teardown():289] cleaning up jupyter logic
2021-10-31 07:05:56,989 INFO    MainThread:397 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 0
2021-10-31 07:05:56,989 INFO    MainThread:397 [wandb_run.py:_restore():1635] restore
2021-10-31 07:05:57,161 INFO    MainThread:397 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 776
  total_bytes: 776
}

2021-10-31 07:05:58,261 INFO    MainThread:397 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 776
  total_bytes: 5202
}

2021-10-31 07:05:58,362 INFO    MainThread:397 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5202
  total_bytes: 5202
}

2021-10-31 07:05:58,463 INFO    MainThread:397 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5202
  total_bytes: 5202
}

2021-10-31 07:05:58,564 INFO    MainThread:397 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5202
  total_bytes: 5202
}

2021-10-31 07:05:58,665 INFO    MainThread:397 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5202
  total_bytes: 5202
}

2021-10-31 07:05:58,766 INFO    MainThread:397 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5202
  total_bytes: 5202
}

2021-10-31 07:05:58,934 INFO    MainThread:397 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5202
  total_bytes: 5202
}

2021-10-31 07:05:59,138 INFO    MainThread:397 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5202
  total_bytes: 5202
}
local_info {
}

2021-10-31 07:06:00,244 INFO    MainThread:397 [wandb_run.py:_append_files():2061] logging synced files
